doc/make.jl

```diff
@@ -336,6 +336,8 @@ end
 
 # Define our own DeployConfig
 struct BuildBotConfig <: Documenter.DeployConfig end
+Documenter.authentication_method(::Documenter.DeployConfig) = Documenter.HTTPS
+Documenter.authenticated_repo_url(::Documenter.DeployConfig) = "https://github.com/JuliaLang/docs.julialang.org.git"
```

Comment:
It actually should be this here (my mistake in the other PR):

```suggestion
Documenter.authentication_method(::BuildBotConfig) = Documenter.HTTPS
Documenter.authenticated_repo_url(::BuildBotConfig) = "https://github.com/JuliaLang/docs.julialang.org.git"
```

The current one works because they become fallback methods.